build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
```

Comment:
I think we need to invent a new MSBuild property, something like /p:TestVersion=23729.  If it is not defined in dir.props, it would be set to some value we choose.  And everywhere we see 23729 (or its equivalent) should become something like:
```
<ExpectedPrerelease>rc3-$(TestVersion)</ExpectedPrerelease>
```
We need to get hardcoded versions out of the files that change frequently and not have the same literal in several places.